configure.ac

```diff
@@ -755,6 +755,12 @@ fi
 # for mono/metadata/debug-symfile.c
 AC_CHECK_HEADERS(elf.h)
 
+# for perf jit dump support
+AC_CHECK_HEADERS(sys/mman.h)
+if test x$ac_cv_header_sys_mman_h = xyes -a  x$ac_cv_header_elf_h = xyes -a x$ac_cv_header_sys_syscall_h = xyes; then
+   AC_DEFINE(ENABLE_JIT_DUMP, 1, [Enable jit dump support on Linux])
+fi
```

Comment:
Do we need a check that the host is Linux?

Reply:
Do sys/mman.h, sys/syscall.h or elf.h exist, if it is not linux?

Reply:
I'm not sure.  It's not really a big deal to leave it on on every platform where the headers exist.

Reply:
I know for a fact that gettid() is Linux-specific.